.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: /acceptance-tests
```

Comment:
We do not have /acceptance-tests yet.
Is it aligned that this will come and we already add it upfront?

Reply:
No. I copied from haproxy-boshrelease... Fair point.